go/stats/statsd/statsd_test.go

```diff
@@ -365,9 +326,7 @@ func TestStatsdGaugesWithSingleLabel(t *testing.T) {
 		if kv.Key == name {
 			found = true
 			sb.addExpVar(kv)
-			if err := sb.statsdClient.Flush(); err != nil {
-				t.Errorf("Error flushing: %s", err)
-			}
+			assert.NoError(t, sb.statsdClient.Flush(), "Error flushing")
 			bytes := make([]byte, 4096)
 			n, err := server.Read(bytes)
 			if err != nil {
```

Comment:
should be a `require.NoError(t, err)` here

Reply:
The added `require.NoError(t, err)` replaces the `if err != nil { ... t.Fatal(err) }` which should now be removed.